crux-http-client/src/crux/remote_api_client.clj

```diff
@@ -219,71 +221,75 @@
           (throw e)))))
 
   (hasTxCommitted [_ submitted-tx]
-    (api-request-sync (str url "/tx-committed?tx-id=" (:crux.tx/tx-id submitted-tx))
-                      {:http-opts {:method :get}
-                       :->jwt-token ->jwt-token}))
+    (->
+     (api-request-sync (str url "/_crux/tx-committed")
+                       {:http-opts {:method :get
+                                    :query-params {:tx-id (:crux.tx/tx-id submitted-tx)}}
+                        :->jwt-token ->jwt-token})
+     (get :tx-committed?)))
 
   (openTxLog [this after-tx-id with-ops?]
-    (let [params (->> [(when after-tx-id
-                         (str "after-tx-id=" after-tx-id))
-                       (when with-ops?
-                         (str "with-ops=" with-ops?))]
-                      (remove nil?)
-                      (str/join "&"))
-          in (api-request-sync (cond-> (str url "/tx-log")
-                                 (seq params) (str "?" params))
+    (let [in (api-request-sync (str url "/_crux/tx-log")
                                {:http-opts {:method :get
-                                            :as :stream}
+                                            :as :stream
+                                            :query-params {:after-tx-id after-tx-id
+                                                           :with-ops? with-ops?}}
                                 :->jwt-token ->jwt-token})]
 
       (cio/->cursor #(.close ^Closeable in)
                     (edn-list->lazy-seq in))))
 
   (sync [_ timeout]
-    (api-request-sync (cond-> (str url "/sync")
-                        timeout (str "?timeout=" (.toMillis timeout)))
-                      {:http-opts {:method :get}
-                       :->jwt-token ->jwt-token}))
+    (->
+     (api-request-sync (str url "/_crux/sync")
+                       {:http-opts {:method :get
+                                    :query-params {:timeout (some-> timeout (cio/format-duration-millis))}}
+                        :->jwt-token ->jwt-token})
+     (get :crux.tx/tx-time)))
 
   (awaitTxTime [_ tx-time timeout]
-    (api-request-sync (cond-> (str url "/await-tx-time?tx-time=" (cio/format-rfc3339-date tx-time))
-                        timeout (str "&timeout=" (cio/format-duration-millis timeout)))
-                      {:http-opts {:method :get}
-                       :->jwt-token ->jwt-token}))
+    (->
+     (api-request-sync (str url "/_crux/await-tx-time" )
```

Comment:
nit: trailing space, and can we have the first arg to `->` on the same line as the `->`?